sci-misc/boinc/boinc-7.9.3.ebuild

```diff
@@ -0,0 +1,196 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER=3.0-gtk3
+
+inherit autotools eutils gnome2-utils linux-info systemd user versionator wxwidgets
+
+MY_PV="7.10"
+
+DESCRIPTION="The Berkeley Open Infrastructure for Network Computing"
+HOMEPAGE="http://boinc.ssl.berkeley.edu/"
+SRC_URI="https://github.com/BOINC/boinc/archive/client_release/${MY_PV}/${PV}.tar.gz -> ${P}.tar.gz
+	X? ( http://boinc.berkeley.edu/logo/boinc_glossy2_512_F.tif -> ${PN}.tif )"
+RESTRICT="mirror"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="X cuda curl_ssl_gnutls curl_ssl_libressl +curl_ssl_openssl static-libs"
+
+REQUIRED_USE="^^ ( curl_ssl_gnutls curl_ssl_libressl curl_ssl_openssl ) "
+
+# libcurl must not be using an ssl backend boinc does not support.
+# If the libcurl ssl backend changes, boinc should be recompiled.
+RDEPEND="
+	!sci-misc/boinc-bin
```

Comment:
Technically blockers should not end up in `DEPEND` (as it is later inheriting `RDEPEND`). I know though you will see this everywhere in tree. For this to fix you would need to s/RDEPEND/COMMON_DEPEND/ in here minus the two blockers, and move the blockers to a separate new `RDEPEND` block.

Reply:
And more than that, there is no sci-misc/boinc-bin package. Further I have no idea what the issue with quickswitch is. Those blockers are really ancient.

I could remove the boinc-bin blocker, but I have to do some research about quickswitch first. Maybe they should both just be kept...

Reply:
Found it:

https://bugs.gentoo.org/210877 : 
"_app-admin/quickswitch-1.05 and sci-misc/boinc-5.10.28 both install /usr/bin/switcher_"

That is still valid, so that blocker must remain.

Reply:
Yes, wherever `sci-misc/boinc-bin` is coming from, it predates portage git migration, so it can safely be dropped.